framework/src/media/utils/MediaUtils.cpp

```diff
@@ -307,14 +307,23 @@ bool header_parsing(FILE *fp, audio_type_t audioType, unsigned int *channel, uns
 	unsigned char *header;
 	unsigned char tag[2];
 	bool isHeader;
+	int ret;
 
 	switch (audioType) {
 	case AUDIO_TYPE_MP3:
 		isHeader = false;
-		while (fread(tag, sizeof(unsigned char), 1, fp) > 0) {
-			if (tag[0] == 0xFF) {
+		while (fread(tag, sizeof(unsigned char), 2, fp) > 0) {
+			/* 11 bits for MP3 Sync Word(the beginning of the frame) */
+			if ((tag[0] == 0xFF) && ((tag[1] & 0xE0) == 0xE0)) {
 				isHeader = true;
 				break;
+			} else {
+				ret = fseek(fp, -1, SEEK_CUR);
```

Comment:
you had read 2 but move back to '-1'..
Isn't '-2' is correct?

Reply:
-1 is correct.
because of,  this case is 0x00 0xFF 0xF3